### PR TITLE
BSP-382: Transforming the rest of the documents from exception record

### PR DESCRIPTION
**Pull request by jvictorcf**
### Change description ###
Implementing the rest of the documents coming from exception record.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java [hunk 1]
```diff
@@ -75,12 +78,66 @@
...
+        )));
+
+        inputScannedDocs.stream()
+            .filter(doc -> doc.getSubtype().equals("DraftConsentOrder"))
+            .findFirst()
+            .map(this::transformInputScannedDocIntoCaseDocument)
+            .ifPresent(doc -> {
+                additionalCaseData.put("consentOrder", doc);
+                additionalCaseData.put("latestConsentOrder", doc);
+            });
```
Comment (shaunganley) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java:105]: indent off

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java [hunk 2]
```diff
@@ -75,12 +78,66 @@
...
+            "CoverLetter", "Letter"
+        )));
+
+        inputScannedDocs.stream()
+            .filter(doc -> doc.getSubtype().equals("DraftConsentOrder"))
+            .findFirst()
+            .map(this::transformInputScannedDocIntoCaseDocument)
+            .ifPresent(doc -> {
+                additionalCaseData.put("consentOrder", doc);
+                additionalCaseData.put("latestConsentOrder", doc);
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java:104]: CCDConsts has LATEST_CONSENT_ORDER

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java [hunk 3]
```diff
@@ -396,8 +398,19 @@ public void shouldTransformAnyAuthorisationSignedToYes() {
     public void shouldTransformScannedDocuments() {
         List<InputScannedDoc> scannedDocuments = new ArrayList<>();
         scannedDocuments.add(InputScannedDoc.builder().subtype("D81").document(new InputScannedDocUrl("http://url/d81-1", "http://binUrl/d81-1/binary", "d81-1.pdf")).build());
-        scannedDocuments.add(InputScannedDoc.builder().subtype("FormA").document(new InputScannedDocUrl("http://url/formA", "http://binUrl/formA/binary", "formA.pdf")).build());
         scannedDocuments.add(InputScannedDoc.builder().subtype("D81").document(new InputScannedDocUrl("http://url/d81-2", "http://binUrl/d81-2/binary", "d81-2.pdf")).build());
+        scannedDocuments.add(createDoc("FormA"));
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java:402]: not "formA"?

File: src/test/resources/fixtures/bulkscan/transformation/simple-formA.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "case_type_id": "FinancialRemedyContested",
   "id": "LV481297",
   "po_box": "PO 17",
-  "form_type": "formA",
+  "form_type": "FormA",
```
Comment (yarpo) [inline src/test/resources/fixtures/bulkscan/transformation/simple-formA.json:5]: why don't we also rename a file to `simple-FormA.json`?

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java (hunk 2 above)
Comment (jvictorcf) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java:104]: Cool. Will change.

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java (hunk 3 above)
Comment (jvictorcf) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java:402]: This is what's specified in the SIT form.

File: src/test/resources/fixtures/bulkscan/transformation/simple-formA.json (hunk 4 above)
Comment (jvictorcf) [reply src/test/resources/fixtures/bulkscan/transformation/simple-formA.json:5]: done

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java (hunk 2 above)
Comment (jvictorcf) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]: done

**Pull request merged**